dev-util/git_delta/git_delta-0.18.3~git.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A syntax-highlighting pager for git, diff, grep, and blame output"
+DESCRIPTION="Delta allows extensive flexibility to the layout and styling of diffs"
+HOMEPAGE="https://dandavison.github.io/delta/"
+COPYRIGHT="2020 Dan Davison"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev=acd758f7a08df6c2ac5542a2c5a4034c664a9ed8
+SOURCE_URI="https://github.com/dandavison/delta/archive/$srcGitRev.tar.gz"
+#future?
+#SOURCE_URI="https://github.com/dandavison/delta/archive/refs/tags/0.18.3.tar.gz"
+SOURCE_DIR="delta-$srcGitRev"
+CHECKSUM_SHA256="546a9a7d405c41c9108242da5875036fbcfbf6d5b3b810c183ee0e2f550ec1ce"
+PATCHES="$portVersionedName.patchset"
+ARCHITECTURES="?all x86_64"
+
+PROVIDES="
+	git_delta = $portVersion
+	cmd:delta = $portVersion
+	"
+REQUIRES="
+	haiku
+	zlib
+	libevent
```

Comment:
These should be sorted alphabetically (except "haiku" and "haiku_devel"). Also, we usually use library names (like `lib:libz`) instead of package names for these.